src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ParserTest {
+
+    private final PrintStream standardOut = System.out;
+    private final ByteArrayOutputStream outputStreamCaptor = new ByteArrayOutputStream();
```

Comment:
Should these constant names be capitalised?

Reply:
I'm not sure, these constants are not static, does it follow the convention to change them to STANDARD_OUT and OUTPUT_STREAM_CAPTOR?

Reply:
Yep I checked on this, will resolve